Automattic/jetpack
# Contact form: more defensive in screen checks, avoid PHP warnings

Fixes #34538

## Proposed changes:

Be more defensive when checking whether we are on the right screen, to avoid PHP warnings.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

I am not quite sure how to reproduce the original issue just yet. We may receive additional information to help with that in #34538.

Until then, you can try the following:

1. Go to Pages > Add New
2. Insert a form in a new page.
3. Publish your page.
4. Access that page in the frontend, and leave a test submission.
5. Go to the Feedback menu in wp-admin.
6. Attempt some of the actions, like marking the submissions as spam; they should work as expected.
